demisto_sdk/commands/unify/tests/yml_unifier_test.py

```diff
@@ -84,6 +85,14 @@ def __init__(self, tenant_id: str = '', auth_id: str = '', enc_key: str = '',
 TESTS_DIR = f'{git_path()}/demisto_sdk/tests'
 
 
+def get_dummy_module(name='MicrosoftApiModule', path=None):
+    if name == 'MicrosoftApiModule':
+        class_name = 'MicrosoftClient'
+    else:
+        class_name = 'CrowdStrikeApiClient'
```

Comment:
today there are only two options, but someone may add a third and get unexpected behavior. I suggest 
```suggestion
    class_name = {'MicrosoftApiModule':'MicrosoftClient', 'CrowdStrikeApiModule':'CrowdStrikeApiClient'}[name]
```